frontend/src/components/global/Nav2.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHome } from '@fortawesome/free-solid-svg-icons';
+import { faUserCog } from '@fortawesome/free-solid-svg-icons';
+import { faPlusSquare } from '@fortawesome/free-solid-svg-icons';
```

Comment:
these 3 lines can be combined into 1 line

Reply:
this has been condensed